go.mod

```diff
@@ -159,7 +159,7 @@ require (
 	github.com/prometheus/mysqld_exporter v0.14.0
 	github.com/prometheus/node_exporter v1.6.0
 	github.com/prometheus/procfs v0.15.0
-	github.com/prometheus/prometheus v0.51.2 // a.k.a. v2.51.2
+	github.com/prometheus/prometheus v1.99.0
```

Comment:
Heads up: this version seems wrong. 1.99.0 was published on April 4, 2022, and has since [been retracted](https://pkg.go.dev/github.com/prometheus/prometheus@v1.99.0).

Reply:
I actually wonder if I need to change that at all since its overridden with a replace anyways. Will test.

Reply:
Ah yeah, it probably doesn't need to change but it might make sense to have it reflect the replace directive as closely as possible (but you're right that it really doesn't matter)

Reply:
Agreed.